src/Chapter2.hs

```diff
@@ -740,7 +758,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Integer -> [[Integer]] -> [[Integer]]
+contains _ [] = []
+contains n (x:xs) = if elem n x == True then x : contains n xs else contains n xs
```

Comment:
`elem n x` is bool itself, so no need to compare it to `True`, just `if elem n x then...`.

On a general note, there is the standard `filter` function, that would do this check for you 🙂